docs/modules/nifi/examples/getting_started/getting_started.sh

```diff
@@ -146,7 +146,8 @@ metadata:
   name: simple-nifi
 spec:
   image:
-    productVersion: 1.27.0
+    productVersion: 2.2.0
+    pullPolicy: Never
```

Comment:
I don't think we should leave that in ;)
```suggestion
```

Reply:
Oops, forgot to remove that one.

I do wish we could just set them all to `IfNotPresent` to aid in local testing :/

Reply:
Removed in a2cc56b548619f9a5a83f2dad67b11d665cd81fd

Reply:
We do that at least for the kuttl tests ;)